app/elements/common/location/location-finder.html

```diff
@@ -0,0 +1,154 @@
+<dom-module id="location-finder">
+	<template>
+		<style>
+			#container {
+				height: 400px;
+				width: 100%;
+			}
+			#map {
+				height: 100%;
+			}
+			.controls {
+				background-color: #fff;
+				border-radius: 2px;
+				border: 1px solid transparent;
+				box-shadow: 0 2px 6px rgba(0, 0, 0, 0.3);
+				box-sizing: border-box;
+				font-family: Roboto;
+				font-size: 15px;
+				font-weight: 300;
+				height: 29px;
+				margin-left: 17px;
+				margin-top: 10px;
+				outline: none;
+				padding: 0 11px 0 13px;
+				text-overflow: ellipsis;
+				width: 400px;
+			}
+
+			.controls:focus {
+				border-color: #4d90fe;
+			}
+		</style>
+		<div id="container">
+			<input id="pacInput" class="controls" type="text" placeholder="Enter a location">
+			<div id="map"></div>
+		</div>
+	</template>
+	<script>
+		Polymer({
+			is: 'location-finder',
+			properties: {
+				selectedLocation: {
+					type: Object,
+					notify: true,
+					reflectToAttribute: true
+				},
+				center: {
+					type: Object,
+					notify: true,
+					value: function() {
+						// If the user does not allow us to center around his location, center the map around Viena.
+						return {
+							lat : 48.215876,
+							lng : 16.389119
+						};
+					}
+				}
+			},
+			observers: [
+				'_recenterMap(center)'
+			],
+			_recenterMap: function(){
+				if (!!this.map){
+					this.map.setCenter(this.center);
+				} else {
+					setTimeout(this.initMap(), 1);
```

Comment:
Why a timeout of 1ms, and not calling it directly? Furthermore `this.initMap()` will be evaluated and it's return value passed on to `setTimeout` which is probably not what you want. You can wrap that call inside an anonymous function.